internal/worker_runner.go

```diff
@@ -389,13 +396,33 @@ func (wr *workerRunner) queryOBD(powerStatus *api.PowerStatusResponse) {
 			}
 		} else {
 			wr.queryOBDWithAP(request, powerStatus)
+			if !wr.signalDumpFramesQ.jobDone {
+				if wr.signalDumpFramesQ.lastEnqueued.Before(time.Now().Add(3 * time.Minute)) {
+					bytes, err := json.Marshal(wr.signalDumpFramesQ.signalFrames)
+					if err != nil {
+						wr.logger.Err(err).Msg("failed to marshal signalDumpFrames")
+					}
+					err = wr.dataSender.SendCanDumpData(bytes)
+					if err != nil {
+						wr.logger.Err(err).Msgf("failed to send canDumpData for custom pids. data length: %d", len(bytes))
+					}
+					wr.logger.Info().Msgf("successfully sent signalDumpFrames. data length: %d", len(bytes))
```

Comment:
should we change it to debug level?

Reply:
should only be once per bootup,

Reply:
I made a change so it is only once  the first time around and then caches it. Since we only set pids slice on start i think this should be ok.

Reply:
Thanks for clarifying it; ah, you use Job to control a one-time run only.